server/schedulers/hot_region_config.go

```diff
@@ -331,3 +370,32 @@ func getWriteLeaderPriorities(c *prioritiesConfig) []string {
 func getWritePeerPriorities(c *prioritiesConfig) []string {
 	return c.writePeer
 }
+
+// adjustConfig will adjust config for cluster with low version tikv
+// because tikv below 5.2.0 does not report query information, we will use byte and key as the scheduling dimensions
+func adjustConfig(cluster opt.Cluster, origins []string, getPriorities func(*prioritiesConfig) []string) []string {
+	querySupport := cluster.IsFeatureSupported(versioninfo.HotScheduleWithQuery)
+	withQuery := slice.AnyOf(origins, func(i int) bool {
+		return origins[i] == QueryPriority
+	})
+	compatibles := getPriorities(&compatibleConfig)
+	if !querySupport && withQuery {
+		schedulerCounter.WithLabelValues(HotRegionName, "use-compatible-config").Inc()
```

Comment:
I think using `debug` log here is more suitable than recorded by metrics.

Reply:
It may be too late to change the log level when you need to confirm the matter.